### PR TITLE
Added comment for PayloadDisco

### PR DESCRIPTION
**Pull request by logarithm**

### CODE REVIEW

File: src/main/scala/com/metamx/common/scala/net/curator/Disco.scala [hunk 1]
```diff
@@ -90,6 +90,12 @@ abstract class AbstractDisco[T](curator: CuratorFramework, config: DiscoConfig,
 
 class Disco(curator: CuratorFramework, config: DiscoConfig) extends AbstractDisco[Void](curator, config)(Void.TYPE)
 
+/**
+ * Please note, that if you use PayloadDisco with not null payload you will get a ClassCastException when you try
+ * to get this data by the Disco class. But you can use PayloadDisco to get data which was written by Disco class.
+ * If you want to update your service's announce from Disco to PayloadDisco, first make sure that all clients
+ * of this service updated to PayloadDisco.
```
Comment (xvrl) [inline src/main/scala/com/metamx/common/scala/net/curator/Disco.scala]: `updated` -> `are updated`

File: src/main/scala/com/metamx/common/scala/net/curator/Disco.scala [hunk 2]
```diff
@@ -90,6 +90,12 @@ abstract class AbstractDisco[T](curator: CuratorFramework, config: DiscoConfig,
 
 class Disco(curator: CuratorFramework, config: DiscoConfig) extends AbstractDisco[Void](curator, config)(Void.TYPE)
 
+/**
+ * Please note, that if you use PayloadDisco with not null payload you will get a ClassCastException when you try
+ * to get this data by the Disco class. But you can use PayloadDisco to get data which was written by Disco class.
```
Comment (xvrl) [inline src/main/scala/com/metamx/common/scala/net/curator/Disco.scala]: `by` -> `using`

Comment (logarithm) [inline src/main/scala/com/metamx/common/scala/net/curator/Disco.scala]: done

File: src/main/scala/com/metamx/common/scala/net/curator/Disco.scala (hunk 1 above)
Comment (logarithm) [inline src/main/scala/com/metamx/common/scala/net/curator/Disco.scala]: done

**Pull request merged**